### PR TITLE
Validation: Fix missing descriptions 

### PR DESCRIPTION
**Pull request by annmariehitchcock**
This PR addresses the following issue:
description-missing

There are 30 issues fixed acros 30 files.

### CODE REVIEW

**Comment by annmariehitchcock** [comment]
Global effort to fix validation errors

@adegeo

The Content & Learning team is fixing build validation errors on docs.microsoft.com. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. I’d very much appreciate your timely review and feedback on these changes I’ve suggested. Thanks!

File: dotnet-desktop-guide/framework/wpf/data/value-xattribute-dynamic-property.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Value (XAttribute dynamic property)
+description: Learn about how the value (XAttribute dynamic property) retrieves or sets the value of the XML attribute.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/data/value-xattribute-dynamic-property.md:3]:
```suggestion
description: Learn about how the XAttribute.Value property retrieves or sets the value of the XML attribute.
```

File: dotnet-desktop-guide/framework/wpf/data/value-xelement-dynamic-property.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Value (XElement dynamic property)
+description: Learn about how the value (XElement dynamic property) retrieves or applies the content of the element.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/data/value-xelement-dynamic-property.md:3]:
```suggestion
description: Learn about how the XElement.Value property retrieves or applies the content of the element.
```

File: dotnet-desktop-guide/framework/wpf/data/xml-xelement-dynamic-property.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Xml (XElement dynamic property)
+description: Learn about how the XML (XElement dynamic property) retrieves the unformatted XML content of the element.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/data/xml-xelement-dynamic-property.md:3]:
```suggestion
description: Learn about how the XElement.Xml property retrieves the unformatted XML content of the element.
```

**Review by adegeo — APPROVED** [review]

**Pull request merged**